### PR TITLE
Create new PhpArray stub

### PR DESCRIPTION
**Pull request by akalongman**
Create a new PhpArray stub and removed old deprecated version

### CODE REVIEW

File: redis/RedisArray.php [hunk 1]
```diff
@@ -0,0 +1,213 @@
+<?php
+
+/**
+ * Helper autocomplete for php redis extension
+ *
+ * @mixin \Redis
+ *
+ * @author Avtandil Kikabidze <akalongman@gmail.com>
```
Comment (DmitryTronin) [inline redis/RedisArray.php:8]: Could you please remove the `@author` and a link to your personal info? No one does this and this information doesn't look suitable here.

**Review by DmitryTronin — CHANGES_REQUESTED** [review]

File: redis/RedisArray.php (hunk 1 above)
Comment (akalongman) [reply redis/RedisArray.php]: No one does? :)
Screenshot from 2022-02-18 18-34-20
Ok, removed

**Review by DmitryTronin — APPROVED** [review]

**Comment by DmitryTronin** [comment]
Could you please update a project & push again? The tests are failing because of recent changes that are not included in this PR.

**Comment by akalongman** [comment]
@DmitryTronin done

**Comment by DmitryTronin** [comment]
Sorry, the tests are still failing with `The commited stub map is not up to date. Please regenerate it using ./generate-stub-map`.
Could you please run `docker-compose -f docker-compose.yml run test_runner /usr/local/bin/php /opt/project/phpstorm-stubs/tests/Tools/generate-stub-map` and commit the resulting `PhpStormStubsMap.php`?

**Comment by akalongman** [comment]
Done

**Pull request merged**